### ISSUE TITLE
Question of closed form factorization, diff augmentation

### ISSUE THREAD

**Issue by JongHwa-Shim** [rosinality/stylegan2-pytorch#198]
I observed several techniques not mentioned in the Stylegan2 paper. (e.g closed form factorization, diff augmentation)
Are these techniques used in the actual original?

**Comment by rosinality** [comment]
No. closed form factorization came from https://arxiv.org/abs/2007.06600, and diff augment (I think what you referring is actually non leaking augmentation/ada) is came from https://arxiv.org/abs/2006.06676.
